README.md

```diff
@@ -10,3 +10,7 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## My Response
+Let $A$ and $B$ both be graphs that have the same number of nodes and are each completely connected.  Then let $f$ be a function that maps $A$ onto $B$ such that each node in $A$ maps to only one node in $B$.  Then this function is both one to one as well as onto which satisfies the need for bijection.  And since both graphs are completely connected there will always be an edge in $A$ that corresponds to an edge in $B$.  
```

Comment:
You have to show that this function exists. Here you're assuming that it does, with the required properties.

Reply:
alright, would I use f: v1 -> v2 for that? To show that it's specifically mapping one vertex from A to one vertex in B?

Reply:
You have to make that more explicit. You could, for example, say that you're mapping the first node in the first graph to the first node in the second graph, the second to the second, and so on.